cli/bootstrap.go

```diff
@@ -98,15 +146,25 @@ var cmdBootstrap = []cobra.Command{
 		},
 	},
 	{
```

Comment:
I tired to test the bootstrap 
But I got not found 

In CLI bootstrap url  is not configured http://localhost 
In nginx there is not route for bootstrap

Reply:
The URL is provided by the `-bootstrap-url` or `-b` flag

Reply:
I think for the longer form of flags, they should have double dashes `--`

Reply:
I think for the longer form of flags, they should have double dashes `--`

Reply:
Is this one resolved, @arvindh123 can you test and confirm?